content/applications/finance/fiscal_localizations/indonesia.rst

```diff
@@ -181,3 +201,31 @@ QR-code` option to `QRIS`.
 
 Ensure that the :guilabel:`Recipient Bank` is the one you configured, as Odoo uses this field to
 generate the QRIS QR code.
+
+POS payment method configurations
+---------------------------------
+
+If you installed Point Of Sale, you can also add a new payment method for PoS via QRIS.
+Go to :menuselection:`Point of Sale --> Configuration --> Payment Methods`, create a new payment
+method.
+
+.. image:: indonesia/indonesia-qris-payment-method.png
+    :alt: QRIS Payment Method configuration
+
+Afterwards, add this payment method the PoS configurations that allow the payment.
+
+.. image:: indonesia/indonesia-qris-pos-config.png
+    :alt: PoS store configuration
+
+Issue QRIS on Point of sale
+---------------------------
+
+On the payment page of Point of Sale, choose the QRIS payment method as configured. Choose
+the amount you want to pay and click on the Send button.
+
+Then, a popup containing a QR code should appear. Click on Confirm Payment to verify the status
+of the payment. When QRIS hasn't successfully received the payment it will issue an error. If
+payment is received, then Point of Sale will be directed to the receipt page.
```

Comment:
```suggestion
Odoo will display a QR code. Click on :guilabel:`Confirm Payment` to verify the status of the
payment. If QRIS hasn't successfully received the payment it will issue an error. If payment is
received, then Point of Sale will be directed to the receipt page.
```